internal/receiver/scriptedinputsreceiver/procpipe/script_others.go

```diff
@@ -0,0 +1,40 @@
+// Copyright Copyright Splunk, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package procpipe
+
+import (
+	_ "embed"
+	"strings"
+)
+
+const includePattern = ". \"$(dirname \"$0\")\"/common.sh"
+
+//go:embed scripts/cpu.sh
+var cpuScript string
+
+//go:embed scripts/df.sh
+var dfScript string
+
+//go:embed scripts/ps.sh
+var psScript string
+
+//go:embed scripts/common.sh
+var commonScript string
+
+var scripts = map[string]string{
```

Comment:
this can be built by calling a new function instead of repeating the Replace call for all scripts